router/xgress/link_receive_buffer.go

```diff
@@ -84,9 +85,9 @@ func (buffer *LinkReceiveBuffer) getLastBufferSizeSent() uint32 {
 	return atomic.LoadUint32(&buffer.lastBufferSizeSent)
 }
 
-func (buffer *LinkReceiveBuffer) Inspect() map[string]interface{} {
-	result := map[string]interface{}{}
-	result["size"] = buffer.Size()
-	result["lastSizeSent"] = buffer.getLastBufferSizeSent()
-	return result
+func (buffer *LinkReceiveBuffer) Inspect() inspect.XgressRecvBufferDetail {
```

Comment:
return `*inspect.XgressRecvBufferDetail` for consistency with `LinkSendBuffer.Inspect()`

Reply:
Fixed